front/src/modules/rollingStock/components/RollingStockEditor/CurveParamSelectors.tsx

```diff
@@ -338,6 +341,16 @@ const CurveParamSelectors = ({
               }}
               dataTestId="power-restriction-selector"
             />
+            {isWarning && (
+              <div className="warning-message text-secondary">
+                <div className="pl-3 d-flex align-items-center">
+                  <div className="mr-2 warning-icon">
+                    <Alert />
+                  </div>
+                  <span>{t('missingPowerClass')}</span>
```

Comment:
You can even pluralize the message: you just need to add a param count in t and in the translation file, you need to add 2 translations `missingPowerClass_one` and `missingPowerClass_other`
```suggestion
                  <span>{t('missingPowerClass', {count: missingPowerClassesNb})}</span>
```

Reply:
Fixed, ty for the tip !